website/docs/icons/usage-guidelines/partials/code/how-to-use.md

```diff
@@ -190,7 +190,23 @@ The component exposes the following _props_:
 
 The size of the icon is determined by the size of the asset imported (each icon is exported in two sizes, _16_ and _24_). If you need a different size, use CSS to override its intrinsic size.
 
-#### Animated icons
+### Aligning icons
+
+By default, the `FlighIcon` component has a `inline-block` display value (this can be changed using the `@isInlineBlock` argument). This means that the icon behaves like an inline element, and that if you want to vertically align it in relation to other sibling elements, you have to use CSS to achieve the expected result.
```

Comment:
```suggestion
By default, the `FlightIcon` component has an `inline-block` display value (this can be changed using the `@isInlineBlock` argument). This means that the icon behaves like an inline element, and that if you want to vertically align it in relation to other sibling elements, you have to use CSS to achieve the expected result.
```